content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -29,22 +29,20 @@ Prisma Client provides the following metrics:
   - `prisma_client_queries_total`: The total number of Prisma Client queries executed.
   - `prisma_datasource_queries_total`: The total number of datasource queries executed (SQL queries in relational databases, and commands in MongoDB).
     - The value returned by `prisma_datasource_queries_total` can be greater than `prisma_client_queries_total`, because some Prisma Client operations create multiple queries.
-  - `prisma_pool_connections_opened_total`: The total number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
+  - `prisma_pool_connections_open`: The total number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
   - `prisma_pool_connections_closed_total`: The total number of pool connections closed.
 
 - Gauges (can increase or decrease):
 
-  - `prisma_pool_connections_open`: The number of currently open pool connections.
+  - `prisma_client_queries_active`: The number of currently active Prisma Client queries.
+  - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
   - `prisma_pool_connections_busy`: The number of currently busy pool connections. These pool connections are currently executing a datasource query.
   - `prisma_pool_connections_idle`: The number of pool connections that are not currently being used. These pool connections are waiting for the next datasource query to run.
-  - `prisma_client_queries_active`: The number of currently active transactions.
-  - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
+  - `prisma_pool_connections_opened_total`: The number of currently open pool connections.
 
 - Histograms (metrics data divided into a collection of values; we call each container in the collection a "bucket"):
 
   - `prisma_client_queries_wait_histogram_ms`: The time waiting for a pool connection for all Prisma Client queries in ms.
-  - `prisma_client_queries_duration_histogram_ms`: The execution time for all executed Prisma Client queries in ms. This includes the time taken to execute all database queries, and to carry out all database engine activities, such as joining data and transforming data to the correct format.
-  - `prisma_datasource_queries_duration_histogram_ms`: The execution time for all executed Datasource queries in ms.
```

Comment:
Why remove these?

Reply:
These two were not included as part of the output. I'll add them back in.